components/badge/badgeStyle.css

```diff
@@ -1,20 +1,20 @@
 .badgeIcon i {
     color: rgb(107, 107, 243);
-    margin: 10px;
+    margin: 30px;
 }
 .badgeIcon span {
     position: absolute;
     font-weight: 800;
-    top: 6px;
-    left: 60px;
+    top: 24px;
+    left: 75px;
     padding: 4px 8px;
```

Comment:
Declare values in rem or em instead of px , so it will be convenient when you add responsiveness to website